help/destinations/catalog/crm/hubspot.md

```diff
@@ -0,0 +1,188 @@
+  * In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or
+  * In the **[!UICONTROL Select target field]** window, choose **[!UICONTROL Select attributes]** category and select from the list of attributes populated from the [[!DNL HubSpot] Properties](https://developers.hubspot.com/docs/api/crm/properties) API.
+
+    Both [!DNL HubSpot] [default properties](https://knowledge.hubspot.com/contacts/hubspots-default-contact-properties) and any custom properties are retrieved for selection as target fields. A few available mappings between your XDM profile schema and [!DNL Hubspot] are shown below:
+
+    | Source Field | Target Field |
+    | --- | --- |
+    | `xdm: person.name.firstName` | `Attribute: firstname` |
+    | `xdm: person.name.lastName` | `Attribute: lastname` |
+    | `xdm: workAddress.street1` | `Attribute: address` |
+    | `xdm: workAddress.city` | `Attribute: city` |
+    | `xdm: workAddress.country` | `Attribute: country` |
+
+An example using these mappings is shown below:
+![Platform UI screenshot example with attribute mappings.](../../assets/catalog/crm/hubspot/mappings.png)
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Log in to the [!DNL HubSpot] website, then navigate to the **[!UICONTROL Contacts]** page to check the audience statuses. This list can be configured to display columns for the custom properties created with the audience name with their value being the audience statuses.
+![HubSpot UI screenshot showing the Contacts page with column headers showing the audience name and cells audience statuses](../../assets/catalog/crm/hubspot/contacts.png)
+
+1. You can alternatively drill down into an individual **[!UICONTROL Person]** page and navigate to the properties displaying the audience name and audience statuses.
+![HubSpot UI screenshot showing the Contact page with custom properties displaying the audience name and audience statuses.](../../assets/catalog/crm/hubspot/contact.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Additional resources {#additional-resources}
+
+Additional useful information from the [!DNL HubSpot] documentation is below:
+* [Authentication methods on HubSpot](https://developers.hubspot.com/docs/api/intro-to-auth)
+* [!DNL HubSpot] uses the [Contacts](https://developers.hubspot.com/docs/api/crm/contacts) and [Properties](https://developers.hubspot.com/docs/api/crm/properties) APIs
```

Comment:
Line 174: rephrase as "HubSpot API references for the Contacts and Properties APIs". Include the links, too.